evm/deploy-contract.js

```diff
@@ -26,8 +26,47 @@ const {
     prompt,
     mainProcessor,
     isContract,
+    addDeploymentOptions,
 } = require('./utils');
 
+function getContractPath(contractName) {
```

Comment:
this is quite unwieldy to maintain. Create a helper method that searches the contract name .sol recursively under the artifacts/contracts folder, for gmp-sdk dep first, and then cgp solidity. This should be a convenient helper for other scripts as well.